UltraStar Play/Assets/Common/Graphics/ImageManager.cs

```diff
@@ -9,7 +9,7 @@ public class ImageManager
 {
     // When the cache has reached the critical size, then unused sprites are searched in the scene
     // and removed from memory.
-    private static readonly int criticalCacheSize = 50;
+    private static readonly int criticalCacheSize = 8;
```

Comment:
how did you come up with this number? That seems a bit low for some scenes with for example 6 different player avatar sprites, golden notes sprite, normal notes sprite, various theme elements,...

Reply:
Ups, that one was only for testing that it works. I have 9 song for testing, so when the 9th cover is loaded, the unused resources are removed and I wanted to trigger this.
Will fix this with the next PR.